### DISCUSSION TITLE
Question: Make multiple Asset Players

### DISCUSSION THREAD

**Discussion by FilipeOS** [bluefireteam/audioplayers#1659] [General]
Hi there,

I need advise on how can I place multiple asset players, any working example?

I have multiple cards (accordion), in same page and I need to put a audio file loaded from assets in each. Can you help?

**Comment by Gustl22** [comment]
You can create a list of AudioPlayers and then initialize the same source via `setSource(AssetSource(path: /*...*/))` each.
In the upcoming release we also introduce (AudioPool)[https://github.com/bluefireteam/audioplayers/blob/main/packages/audioplayers/lib/src/audio_pool.dart] which simplifies this process.
